tools/data-model-exporter/data_model_exporter/dmExporter.py

```diff
@@ -0,0 +1,164 @@
+"""!/usr/bin/env python"""
```

Comment:
This shebang should not be commented out

Reply:
I think you'll need to set +x on this file so it's executable from the shell (i.e, so you want have to do a `python dmExporter.py ...` every time but instead `./dmExporter.py`)